ThroughPut/TP_BasePlate_Assembly.gscript

```diff
@@ -101,6 +103,12 @@ END
  
     ADD $bp_target_pos $pcb_pos $top_offset
 
+    # Extra head rotation mitigation between UNL and FNAL.
+    GOTOIF @PCB_ROT_NO_CHANGE `$pcb_rot>$neg45`
```

Comment:
Why use a variable here instead of just `$pcb_rot>-45`?

Reply:
Comparisons with negative numbers seem to crash the script on my end. I tried both $pcb_rot>-45 and -45<$pcb_rot.

Reply:
Ok, then that's probably a bug in gScript itself. I'll have a look at it shortly and push a release with the fix.